src/databricks/labs/remorph/deployment/installation.py

```diff
@@ -25,8 +25,9 @@ def __init__(
         self._recon_deployment = recon_deployment
 
     def install(self, config: RemorphConfigs):
+        wheel_paths: list[str] = []  # self._upload_wheel()
```

Comment:
PR #777 consists the `upload wheel` and `Upgrades` code. This is stacked PR after this.